itests/marathon_python.feature

```diff
@@ -2,6 +2,7 @@ Feature: marathon-python can create and list marathon apps
 
   Scenario: Trivial apps can be deployed
     Given a working marathon instance
+    Then we get the marathon instance's info
```

Comment:
Yes please. Plus once we have a place for that sort of thing we can add other general information gathering methods, (`get_leader`, `get_metrics`, `ping`, etc)